dotnet/src/Connectors/Connectors.Sqlite.UnitTests/SqliteMemoryStoreTests.cs

```diff
@@ -0,0 +1,680 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Connectors.Sqlite;
+using Microsoft.SemanticKernel.Memory;
+using Xunit;
+
+namespace SemanticKernel.Connectors.Sqlite.UnitTests;
+
+/// <summary>
+/// Unit tests of <see cref="SqliteMemoryStore"/>.
+/// </summary>
+[Collection("Sequential")]
+public sealed class SqliteMemoryStoreTests : IDisposable
```

Comment:
Did you just move this?  Don't want to review if I it just a move. 😄

Reply:
Yes, this file is located in `Connectors.UnitTests` project which includes unit tests for multiple connectors. Taking into account that we are creating separate unit test project for each connector, I moved this file to new `Connectors.Sqlite.UnitTests` project. I hope that in the future we will be able to remove `Connectors.UnitTests` completely.